src/components/SignInButtons/AppleSignIn/index.ios.js

```diff
@@ -46,6 +46,7 @@ const AppleSignIn = () => {
     };
     return (
         <ButtonBase
+            accessibilityLabel="common.signInWithApple"
```

Comment:
I was a little confused by this at first because it kind of looks like the screenreader is going to read out "common.signInWithApple" literally.

I think the cleanest implementation of this might be to redo ButtonBase such that it takes a `provider` (or similar) prop that can pick between `"apple"` or `"google"` and then can contain the icon and label 🤔 

At the very least, if we want to take an approach like it currently is, I think we should do the translation where the i18n key is referenced (e.g., this line).

Reply:
That makes sense. It can also take advantage of the CONST values we have set up.